nix/lib.nix

```diff
@@ -80,10 +80,11 @@
 
         # Only consider files that contain exactly one identity, since files with multiple identities are allowed,
         # but are ambiguous with respect to the pairings between identities and pubkeys.
-        if [[ $(grep -c "^AGE-" "$file") == 1 ]]; then
-          if grep -q "^AGE-PLUGIN-YUBIKEY-" "$file"; then
+        file_path=$(realpath "$file")
```

Comment:
> The question I should have been asking is: Why were the extra layer of quotes added in #40? 

The reason is it solved https://github.com/oddlama/agenix-rekey/issues/39 .

But it doesn't seem to work well with the current codebase. Maybe we should revert that pr and just simply disable the shell check which directly caused the error, instead of trying to bypass it.

Reply:
Yeah I think in this case we are aware of that not quoting is the correct thing to do, so I guess disabling the check with a little comment would be the best was forward.